puppet/modules/web/files/deploy-stagingyum.sh

```diff
@@ -0,0 +1,11 @@
+  # Make sure target dir can be created
+  YUM_PATH=`echo "${SSH_ORIGINAL_COMMAND}" | awk '{ print $NF }'`
+  PROJECT=`echo $YUM_PATH | /bin/cut -f2 -d/`
+  RELEASE=`echo $YUM_PATH | /bin/cut -f3 -d/`
+  mkdir -p <%= @home %>/rsync_cache/$PROJECT/$RELEASE
```

Comment:
this file needs to be deployed as a template, or this line not to have erb in it ;-)